client/src/App.js

```diff
@@ -1,87 +1,91 @@
-import React, { Component } from 'react';
-import axios from 'axios'
-import { Route } from 'react-router-dom'
+import React, { Component } from "react";
+import axios from "axios";
+import { Route } from "react-router-dom";
 // components
-import Signup from './components/sign-up'
-import LoginForm from './components/login-form'
-import Navbar from './components/Nav/index'
-import Home from './components/Home/home'
-import CreateJob from './components/CreateJob/createjob'
-import AddChild from './components/AddChild/addchild';
+import Signup from "./components/sign-up";
+import LoginForm from "./components/login-form";
+import Navbar from "./components/Nav/index";
+import Home from "./components/Home/home";
+import CreateJob from "./components/CreateJob/createjob";
+import About from "./components/About/about";
+import AddChild from "./components/AddChild/addchild";
+import Shop from "./components/Shop/shop";
+import Button from "react-bootstrap/Button";
+import MyVerticallyCenteredModal from "./components/Modals/index";
 
 class App extends Component {
-  constructor() {
-    super()
+  constructor(...args) {
```

Comment:
could this just be args, and then the next line have it ...args? I dont think you need ...args, but i may be wrong not sure.

Reply:
Removed arguments, they are useless.